### ISSUE TITLE
Code review summary content lost

### ISSUE THREAD

**Issue by codeluggage** [microsoft/vscode-pull-request-github#3931]
Type: <b>Bug</b>

In an 11 file code review with about 15 comments, trying to write in the split pull request view "Leave a review summary content" text box, seems to quietly refresh the tab and lose the written content. 

I seem to remember this being cached OK in the past. Now I've lost it 3 times.

Extension version: 0.50.0
VS Code version: Code 1.71.0 (784b0177c56c607789f9638da7b6bf3230d47a8c, 2022-09-01T07:25:25.516Z)
OS version: Darwin arm64 21.6.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Max (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|7, 4, 4|
|Memory (System)|32.00GB (2.09GB free)|
|Process Argv|--crash-reporter-id fca85713-53a0-48df-b835-e12ae3c05cef|
|Screen Reader|no|
|VM|0%|
</details>

**Comment by alexr00** [comment]
@codeluggage is this in view in the sidebar or the full description view in the editor?

**Comment by codeluggage** [comment]
@alexr00 This was in 2 editor split view with the (full description) view in the one split and code I was reviewing in the other.

I haven't done that exact workflow much lately, but what I have done hasn't had any issues since. I've been on `1.71.1`.

**Comment by alexr00** [comment]
Thanks for confirming. I tried to repro the issue, but I haven't seen it happen yet. I'm closing this issue for now, but if you see it again I'll re-open and investigate further.

**Comment by mothepro** [comment]
This happens to me all the time I switch focus.
Massive headache and time waster.

**Comment by alexr00** [comment]
I still can't reproduce this. Are there exact steps that always cause it to occur? If so, could you share a screen recording?

**Comment by faenuccio** [comment]
I am experiencing the same problem. I try to explain all the steps in details.

1. I open a PR and make several suggestions to several files, each open in its own editor, the first time clicking on "Start review" and then each time by "Add Review Comment".
2. Once done, I click on the left panel -> the title of the PR to open a new editor containing all the PR information. All my comments are at the bottom, but still unsubmitted. The "submit Review" button is clickable, below an empty field with (greyed) the description "Leave a review summary comment"
3. I start typing something in this review comment.
4. Then I switch to one of the open editors with one of the reviewed files (typically, if I'm commenting "I find that the function xxx in file yyy has problem PPP" and I forgot some detail, I want to double-check the file before writing this down).
5. When I switch back to the PR editor to continue my review comment, all the text typed in point 3. is gone and I need to start from scratch.

I'm on VSCode 1.134.0, with version 0.162.0 of the GitHub PR extension, on Ubuntu 24.04.4 LTS

**Comment by alexr00** [comment]
> When I switch back to the PR editor to continue my review comment, all the text typed in point 3. is gone and I need to start from scratch.

How do you do this step? When I just click back on the tab, the text I typed is there.

**Comment by faenuccio** [comment]
> > When I switch back to the PR editor to continue my review comment, all the text typed in point 3. is gone and I need to start from scratch.
> 
> How do you do this step? When I just click back on the tab, the text I typed is there.

Yeah, I just click on the tab (as I do to switch to any other file).
